routers/bucket/actions/index.js

```diff
@@ -62,14 +63,17 @@ module.exports = ({actions, storage, ...config}) => {
         res.send(204);
       })
       .catch((err) => {
-        res.send(err.message);
```

Comment:
if (err instanceof ....) {
  // handle case for this class, let's say it's not found => 404
} else if (err instanceof ....) {
  // other error, other err code
}